.github/workflows/ci.yml

```diff
@@ -66,25 +47,75 @@ jobs:
         working-directory: core/tests/wasm_tests
 
       - name: Run cargo check
-        run: |
-          cargo check -v --all-features \
-            -p marine-core \
-            -p marine \
-            -p fluence-app-service \
-            -p marine \
-            -p mrepl
+        run: cargo check --all-features
+
+      - name: Run cargo clippy
+        run: cargo clippy --all-features
```

Comment:
I suggest running clippy/check/fmt  after cargo test, because this way developer will see failed tests even when forgot to do fmt / fix clippy warnings. Or just do it in a separate workflow.